package.json

```diff
@@ -14,6 +14,7 @@
     "url": "https://github.com/martysweet/cfn-lint/issues"
   },
   "dependencies": {
+    "@types/js-yaml": "^3.12.0",
```

Comment:
Does this need to be a dependency or should it be a devDependency has it previously was?